src/components/Map/TileChartMap/TileMap/index.js

```diff
@@ -475,7 +475,7 @@ class TileMap {
 
     options.chart.marginRight = 0;
     options.chart.marginLeft = 0;
-    options.chart.marginTop = legendHeight;
+    options.chart.marginTop = legendHeight + 15;
```

Comment:
Should this be set [here](https://github.com/cfpb/ccdb5-ui/blob/main/src/components/Map/TileChartMap/TileMap/index.js#L460+L466) instead?

Reply:
I assumed `legendHeight` was used to control the height of the legend but it looks like the value is only used to set a buffer between the map and the legend. So I think you're right, it probably makes more sense to alter the breakpoints array. 👍

Reply:
I ended up just increasing the margin at all the breakpoints because IMHO the map is too close to the legend at most sizes.